src/Markup/Avalonia.Markup.Xaml/AvaloniaXamlLoaderPortableXaml.cs

```diff
@@ -127,7 +127,19 @@ public object Load(Uri uri, Uri baseUri = null, object rootInstance = null)
 
             using (var stream = assetLocator.Open(uri, baseUri))
             {
-                return Load(stream, rootInstance, uri);
+                try
+                {
+                    return Load(stream, rootInstance, uri);
+                }
+                catch (Exception e)
+                {
+                    var uriString = uri.ToString();
+                    if (!uri.IsAbsoluteUri)
+                    {
+                        uriString = new Uri(baseUri, uri).AbsoluteUri;
+                    }
+                    throw new Exception("Error loading xaml at " + uriString, e);
```

Comment:
Can we use a more descriptive exception type? Like maybe `XamlLoadException` or something?